source/extensions/network/dns_resolver/getaddrinfo/getaddrinfo.cc

```diff
@@ -102,29 +102,25 @@ class GetAddrInfoDnsResolver : public DnsResolver, public Logger::Loggable<Logge
     }
 
     std::list<DnsResponse> final_results;
-    switch (query.dns_lookup_family_) {
+    switch (query.dns_lookup_family_)
     case DnsLookupFamily::All: {
```

Comment:
Drive-by comment from Google's envoy security review: The opening brace here after the first case label, instead of immediately after the `switch (...)`, is surprising! I didn't know C++ allowed this. It seems to work fine, but you might want to move it up a line to avoid confusion.